notebooks/week2/02_04_train_log_custom_model.py

```
@@ -0,0 +1,169 @@
+model_version = mlflow.register_model(
+    model_uri=f'runs:/{run_id}/pyfunc-power-consumption-model',
+    name=model_name,
+    tags={"git_sha": f"{git_sha}"})
+# COMMAND ----------
+
+with open("model_version.json", "w") as json_file:
+    json.dump(model_version.__dict__, json_file, indent=4)
+
+# COMMAND ----------
+model_version_alias = "the_best_model"
+client.set_registered_model_alias(model_name, model_version_alias, "1")  
```

Ruff

Trailing whitespace

Remove trailing whitespace

(W291)